README.md

```diff
@@ -0,0 +1,82 @@
+![Wagtail](https://releases.wagtail.io/wagtail-github-header.png?x)
+
+Wagtail is an open source content management system built on Django, with a strong community and commercial support. It's focused on user experience, and offers precise control for designers and developers. 
+
+![Wagtail screenshot](https://releases.wagtail.io/wagtail-screenshot-with-browser.png)
+
+### Features
+
+* A fast, attractive interface for authors
+* Complete control over design and front-end
+* Scales to millions of pages and thousands of editors
+* Fast out of the box, cache-friendly when you need it
+* Content API for 'headless' sites with de-coupled front-end
+* Runs on a Raspberry Pi or a multi-datacenter cloud platform 
+* StreamField encourages flexible content without compromising structure
+* Powerful, integrated search, using Elasticsearch or PostgreSQL
+* Excellent support for images and embedded content
+* Multi-site and multi-language ready
+* Embraces and extends Django
+
+Find out more at [wagtail.io](https://wagtail.io/).
+
+### Getting started
+
+Wagtail works with Python 3, on any platform.
+
+```
+    pip install wagtail
+    wagtail start mysite
+    cd mysite
+    python manage.py migrate
+    python manage.py createsuperuser
+    python manage.py runserver
+```
+
+For detailed installation and setup docs, see [docs.wagtail.io](http://docs.wagtail.io/).
+
+### Who’s using it?
+
+Wagtail is used by NASA, Google, Oxfam, the NHS, Mozilla, MIT, the Red Cross, Salesforce, NBC, BMW, the US and UK governments. Add your own Wagtail site to [madewithwagtail.org](http://madewithwagtail.org).
+
+### Documentation
+
+[docs.wagtail.io](http://docs.wagtail.io/) is the full reference for Wagtail, and includes guides for developers, designers and editors, alongside release notes and our roadmap.
+
+### Compatibility
+
+Wagtail supports Django 1.11.x and 2.0 on Python 3.4, 3.5 and 3.6. Supported database backends are PostgreSQL, MySQL and SQLite.
+
+---
+
+### Community Support
+
+There is an active community of Wagtail users and developers responding to questions on [Stack Overflow](http://stackoverflow.com/questions/tagged/wagtail). When posting questions, please read Stack Overflow's advice on [how to ask questions](http://stackoverflow.com/help/how-to-ask) and remember to tag your question "wagtail".
+
+For topics and discussions that do not fit Stack Overflow's question and answer format, we have a [Slack workspace](https://github.com/wagtail/wagtail/wiki/Slack) and a [Wagtail Support mailing list](https://groups.google.com/forum/#!forum/wagtail).
```

Comment:
Referral the `#support` channel on Slack would be relevant.

Reply:
The wiki page linked here (which you have to read to get the access URL...) mentions `#support` and other points of etiquette, so I think this is covered.